hack/generate.sh

```diff
@@ -0,0 +1,11 @@
+#!/bin/bash
+
+set -o errexit
+set -o pipefail
+set -o nounset
+
+IN=origin
+for version in v5.5 v5.6; do
+  cp -r $IN $version
```

Comment:
Here you should prune $version directory before copying there. It's necessary when you decide to remove a file from the origin directory.